ldap2pg/config.py

```diff
@@ -386,6 +387,7 @@ class Configuration(dict):
         Mapping('ldap:binddn', env=['LDAPBINDDN', 'LDAP_BIND']),
         Mapping('ldap:user'),
         Mapping('ldap:password', secret=True),
+        Mapping('ldap:referrals', env=['REFERRALS']),
```

Comment:
You should use `LDAPREFERRALS`  to match libldap env vars. By default, `ldap2pg` computes an env var from YAML path with `_` like `LDAP_REFERRALS`. (see env= for ldap:binddn above).

If you don't set env= in the Mapping object for referals, the `ldap2pg.ldap` gather_options functions will anyway gather `LDAPREFERRALS` from env and `REFERRALS` from ldaprc.

Yes, this mean that ldap2pg reads once `LDAP_REFERRALS` (from the env to YAML loop) and then `LDAPREFERRALS` (in the env to ldaprc loop).

Reply:
`Mapping` accepts a `processor` argument. If you pass it `processor=int`, it will ensure `REFERRALS` is an integer.